modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -188,6 +188,7 @@ export class Wallet implements IWallet {
       'numBlocks',
       'nonce',
       'preview',
+      'receiveAddress',
```

Comment:
Is this documented for user visibility in the developer portal?

Reply:
@mmcshinsky-bitgo We are yet to make any changes to the API docs related to Eth TSS Receive Addresses. Will keep track of this and accomodate it when we make changes to api docs
cc: @wesgraham